### PR TITLE
MINOR: update docs to add note about removing SNAPSHOT from streams dependency

### PR DESCRIPTION
**Pull request by dguy**

### CODE REVIEW

File: docs/streams/tutorial.html [hunk 1]
```diff
@@ -62,8 +62,8 @@
     </pre>
 
     <p>
-        The <code>pom.xml</code> file included in the project already has the Streams dependency defined,
-        and there are already several example programs written with Streams library under <code>src/main/java</code>.
+        You will need to update the <code>pom.xml</code> file included in the project to remove SNAPSHOT from the defined Streams dependency.
```
Comment (miguno) [inline docs/streams/tutorial.html]: Suggestion:

```
<strong>Important:</strong> You must manually update the setting of <code>&lt;kafka.version&gt;</code> in the generated <code>pom.xml</code> file from <code>0.11.0.1-SNAPSHOT</code> to <code>0.11.0.1</code>.
```

**Review by miguno — APPROVED** [review]

File: docs/streams/tutorial.html [hunk 2]
```diff
@@ -62,8 +62,8 @@
     </pre>
 
     <p>
-        The <code>pom.xml</code> file included in the project already has the Streams dependency defined,
-        and there are already several example programs written with Streams library under <code>src/main/java</code>.
+        <strong>Important:</strong> You must manually update the setting of <code>&lt;kafka.version&gt;</code> in the generated <code>pom.xml</code> file from <code>0.11.0.1-SNAPSHOT</code> to <code>0.11.0.1</code>.
```
Comment (ijuma) [inline docs/streams/tutorial.html:65]: Maybe worth mentioning that this has been improved/fixed for the next release? That way people won't think we made it hard on purpose. :)

**Review by ijuma — APPROVED** [review]
LGTM, thanks.

**Comment by dguy** [comment]
merged to 0.11.0

**Pull request closed without merging**